src/Ocelot.Provider.Kubernetes/Kube.cs

```diff
@@ -35,17 +33,14 @@ public virtual async Task<List<Service>> GetAsync()
             .ResourceClient(client => new EndPointClientV1(client))
             .GetAsync(_configuration.KeyOfServiceInK8s, _configuration.KubeNamespace);
```

Comment:
The integrated endpoint is capable of generating subsets of varying lengths, which suggests a review of the design may be beneficial.